lib/dao.js

```diff
@@ -304,11 +304,22 @@ DataAccessObject.create = function (data, options, cb) {
           });
         }
 
-        if (connector.create.length === 4) {
-          connector.create(modelName, this.constructor._forDB(val), options, createCallback);
-        } else {
-          connector.create(modelName, this.constructor._forDB(val), createCallback);
-        }
+        var inst = this;
```

Comment:
I believe `this === obj`, in which case I am proposing to drop `inst` and use `obj` instead. Thoughts?